src/classes/Todo.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * To ensure that we don't link different components together
+ * we create "shared" classes in a separate classes-folder, such
```

Comment:
`common` might be good (`shared` is used in full-stack node projects with code that is shared between the backend and the frontend)